### ISSUE TITLE
[BUG] When reviewing kanji cards, error after deleting card via mark known/delete

### ISSUE THREAD

**Issue by saxoncameron** [migaku-official/Migaku-Kanji-Addon#152]
During my reviews. I deleted a card via the button on the back: "Mark known and delete". I continued my reviews, and then after marking a later card as correct, I got the error below. I believe this is linked to the card that I attempted to delete earlier, because when I restarted Anki and continued my reviews, the card I tried to delete was back in my review queue. I deleted it again, continued my reviews, and again later (presumably when that same card was somehow/perhaps at the top of the review queue?) the same error message popped up.

```
Error 
An error occurred. Please start Anki while holding down the shift key, which will temporarily disable the add-ons you have installed. 
If the issue only occurs when add-ons are enabled, please use the Tools>Add-ons menu item to disable some add-ons and restart Anki, repeat until you discover the add-on that is causing the problem. 
When you've discovered the add-on that is causing the problem, please report the issue on the add-ons section of our support site. 
Debug info:
Anki 2.1.49 (dc80804a) Python 3.8.6 Qt 5.14.2 PyQt 5.14.2
Platform: Mac 10.16
Flags: frz=True ao=True sv=2
Add-ons, last update check: 2022-01-09 14:40:24
Add-ons possibly involved: ⁨Migaku Anki Extension⁩

Caught exception:
Traceback (most recent call last):
  File "aqt/reviewer.py", line 464, in <lambda>
  File "/Users/saxoncameron/Library/Application Support/Anki2/addons21/Migaku Anki Extension/reviewer.py", line 93, in reviewer_mod_answer_ease
    Reviewer_answerCard(self, ease)
  File "aqt/reviewer.py", line 422, in _answerCard
  File "anki/scheduler/v2.py", line 463, in answerCard
  File "anki/cards.py", line 117, in flush
  File "anki/_backend/generated.py", line 936, in update_cards
  File "anki/_backend/__init__.py", line 131, in _run_command
anki.errors.NotFoundError
```

**Comment by saxoncameron** [comment]
This is still happening for me on the latest version of everything.

- Anki `2.1.49`
- Migaku Kanji GOD beta.12
- macOS Monterey 12.1

**Comment by RicBent** [comment]
Fixed with f818d8a725974604eb9621fda44cf393458e5aa6
